packages/default/scss/fab/_layout.scss

```diff
@@ -122,6 +122,17 @@
         outline: none;
     }
 
+    // Disabled state
+    .k-fab-item.k-state-disabled,
```

Comment:
Not all elements have `:disbled` pseudo class. As of Nov 2020, it applies onyl to native form elements like `button` and `input`. See https://dojo.telerik.com/@joneff/emeSETEf

As a general rule of thumb, if the element allows for it, prefer placing pseudo `:disabled` selector first, then the attribute `[disabled]` selector and then the rest of the classnames.